src/node.cc

```diff
@@ -277,22 +277,17 @@ MaybeLocal<Value> StartExecution(Environment* env, StartExecutionCallback cb) {
 
   if (cb != nullptr) {
     EscapableHandleScope scope(env->isolate());
+    // TODO(addaleax): pass the callback to the main script more directly,
+    // e.g. by making StartExecution(env, builtin) parametrizable
+    env->set_embedder_mksnapshot_entry_point(std::move(cb));
```

Comment:
Since it's going to be reused I guess we can just drop the `mksnapshot` part and call this `embedder_entry_point`? Or just `custom_entry_point` because in the case of SEA there's not necessarily an embedder..

Reply:
Sure, done in a0e1149ca86a3259381fbfc3a8186960f431e9a8. I think `embedder_entry_point` is fine, the SEA is essentially just an embedder that happens to consistent of the standard binary and extra code.